conf.yaml

```diff
@@ -73,10 +73,10 @@ contents_title:     Welcome to Elastic Docs
 #   <key>: &<variable> <value>
 # The keys don't really matter, but by convention the are the same as the variable.
 variables:
-  stackcurrent: &stackcurrent 8.4
```

Comment:
I think we need to have separate PRs for adding the 8.5 branches to the various books (should be merged now or soon-ish) with actually making this version "current" (should happen during the release).